test/Twig/Tests/Node/EmptyTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+class Twig_Tests_Node_EmptyTest extends Twig_Test_NodeTestCase
+{
+    /**
+     * @expectedException LogicException
+     */
+    public function testAccessingNameThrowsException()
+    {
+        $body = new Twig_Node_Empty();
+        $node = new Twig_Node_Block('foo', $body, 1);
```

Comment:
I saw this in some of the other tests I opened, that's why I copied it. I will remove it, if it's not necessary.